collector/pom.xml

```diff
@@ -38,6 +38,11 @@
       <version>4.11</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
That's the shadowing so that the agent doesn't have the same package names as the code. It'll need changing to org.yaml.